soccer/gameplay/tactics/positions/submissive_goalie.py

```diff
@@ -91,7 +91,8 @@ def block_line(self, value):
 
         self._move_target.x = min(
             max(self._move_target.x,
-                -SubmissiveGoalie.MaxX), SubmissiveGoalie.MaxX)
+                -SubmissiveGoalie.MaxX + 0.01), SubmissiveGoalie.MaxX - 0.01)
```

Comment:
This file looks good, although I'm not sure why we need the +0.01. Could you explain that to me again?  I didn't understand it too well during the meeting.